.github/workflows/makefile.yml

```
@@ -6,12 +6,18 @@ on:
   workflow_dispatch:
 
   push:
+    branches:
+      - main
+      - 'release-\d.\d\d'
```

actionlint (1.7.12)

[error] character '\' is invalid for branch and tag names. only special characters [, ?, +, *, \, ! can be escaped with \. see `man git-check-ref-format` for more details. note that regular expression is unavailable. note: filter pattern syntax is explained at https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet

(glob)